packages/@okta/vuepress-site/docs/guides/configure-ciba/main/index.md

```diff
@@ -0,0 +1,322 @@
+
+**Example request**
+
+```bash
+curl --request POST \
+  --url https://${yourOktaDomain}/oauth2/default/v1/bc/authorize \
+  --header 'Accept: application/json' \
+  --header 'authorization: Basic MG9hY...' \
+  --header 'Content-Type: application/x-www-form-urlencoded' \
+  --data 'scope=openid email'
+  --data 'binding_message=Test Binding Message'
+  --data 'login_hint=test.user@email.com'
+```
+
+**Example response**
+
+```json
+{
+  "auth_req_id": "ftJwF5ZwW2SGPPoTQEKtAr_U8_Ek3RvWyR",
+  "expires_in": 300,
+  "interval": 5
+}
+```
+
+Note the response parameters that are passed:
+
+* `auth_req_id` is a unique identifier that identifies the authentication request made by the client. This parameter must be used in the request by the client to the token endpoint.
+* `expires_in` is the expiration time of the `auth_req_id` in seconds. 300 seconds is the default.
+* `interval` is the minimum amount of time (in seconds) that the client should wait between polling requests to the token endpoint. The default is five minutes.
+
+### Send a request to the token endpoint
+
+The next step in the CIBA authentication flow is to send a request for tokens to the `/token` endpoint.
+
+> **Note:** Currently `poll` is the only supported [backchannel token delivery method](/docs/reference/api/apps/#settings-10).
+
+**CIBA request parameters**
+
+| Parameter                    | Value                                                                        |
+| :--------------------------- | :--------------------------------------------------------------------------- |
+| `grant_type`| `urn:openid:params:grant-type:ciba` |
+| `auth_req_id` | Required if the `grant_type` is `urn:openid:params:grant-type:ciba`. |
+| `scope` | This is a list of scopes that the client wants included in the access token. |
+| `request` | If you are using `private_key_jwt` as the [client authentication method](/docs/reference/api/apps/#credentials), this is the JWT created by the client that enables you to pass requests as a single, self-contained parameter. See [Parameter details](/docs/reference/api/oidc/#client-initiated-backchannel-authentication-parameter-details). |
+
+1. Send a request for tokens to the `/token` endpoint:
+
+   **Example request**
+
+   ```bash
+     curl --request POST \
+     --url https://${yourOktaDomain}/oauth2/default/v1/token \
+     --header 'Accept: application/json' \
+     --header 'authorization: Basic MG9hY...' \
+     --header 'Content-Type: application/x-www-form-urlencoded' \
+     --data 'grant_type=urn:openid:params:grant-type:ciba&scope=openid email&auth_req_id=ftuPCF1eZ0ujBBC3RlRPwTXucFfFQ1M3bh&login_hint=test.user@email.com'
+   ```
+
+   > **Note:** For additional request examples using different parameters as well as a public key/private key [client authentication](/docs/reference/api/oidc/#client-authentication-methods) example, see [CIBA request examples](/docs/reference/api/oidc/#ciba-request-examples).
```

Comment:
Again, didn't see ciba request examples. Link went to: /docs/reference/api/oidc/#possible-errors

Reply:
Yes, this is in Mymoona's CIBA PR https://github.com/okta/okta-developer-docs/pull/3709